Pd 391 user feedback tutorial (#2014)
truenas/documentation: content/SCALE/SCALEReleaseNotes.md

* Storage changes
* Apps changes

> * New Web UI feedback system: A new feedback reporting option has been introduced to encourage  user experience comments and suggestions for screens undergoing revision in prerelease software versions.

Reviewer: how about this:
```suggestion
* New web UI feedback system: A new feedback-reporting option to encourage  user experience comments and suggestions for screens undergoing revision in pre-release software versions.
```
